config/proto_conversions.go

```diff
@@ -88,14 +88,11 @@ func ComponentConfigToProto(component *Component) (*pb.ComponentConfig, error) {
 		return nil, errors.Wrap(err, "failed to convert service configs")
 	}
 
+	// TODO(adam): Should we keep this.
```

Comment:
Should we be doing this fixing during the proto conversions?

Reply:
personally I think no

Reply:
Ack. I removed my comment.